.github/changelog.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+m_branch=m;
+changelog_file=CHANGELOG.md;
+
+# fetch master since we might be in a shallow clone
+git fetch origin "$m_branch:$m_branch" --depth=1
+
+changed=0;
+for log in */"$changelog_file"; do
```

Comment:
I believe you need to remove the "*/" here. Or this cycle altogether, since there is only one changelog file in root.

The ideal would be to replace it with `**` so it would work for any directory depth, but for some stupid reason globstar is turned off by default in bash and I don't know how to set bash options in github CI.

Reply:
Ah right. It would be nice to have a script that works both with one-project repos and multi-project repos... Tho having two versions of the script is not too bad

Reply:
Updated the script